### ISSUE TITLE
E1111 False Positive With Generators

### ISSUE THREAD

**Issue by ryannathans** [PyCQA/pylint#2400]
### Steps to reproduce

```
class Test:

	def __init__(self):
		self.generator = self._make_gen()

	def _make_gen(self):
		a = yield
		while True:
			print(a)
			a = yield
```

### Current behavior

Highlights `self.generator = self._make_gen()` with E1111

### Expected behavior

No E1111 false positive on this line

### pylint --version output

```
__main__.py 2.0.1
astroid 2.0.1
Python 3.6.3 (v3.6.3:2c5fed8, Oct  3 2017, 17:26:49) [MSC v.1900 32 bit (Intel)]
```

**Comment by PCManticore** [comment]
Thanks, I can reproduce this issue!

**Comment by ryannathans** [comment]
Probably because the yield isn't at the start of the line

**Comment by PCManticore** [comment]
This is fixed in the linked commit, I'm doing a bug fix release for `astroid` now. Should be available in ~10 minutes.
